src/components/settings/Recovery/index.tsx

```diff
@@ -86,11 +86,11 @@ export function Recovery(): ReactElement {
             },
             [HeadCells.TxCooldown]: {
               rawValue: txCooldownSeconds,
-              content: <Typography>{getPeriod(txCooldownSeconds)}</Typography>,
+              content: <Typography>{txCooldownSeconds === 0 ? 'none' : getPeriod(txCooldownSeconds)}</Typography>,
```

Comment:
The minimum delay value is 60 afaik. How can this case happen?

Reply:
On a contractual level, the `txCooldown` [can be any value](https://github.com/gnosis/zodiac-modifier-delay/blob/main/contracts/Delay.sol#L87-L93). (`txExpiration` is [limited to `0` or `>=60` seconds](https://github.com/gnosis/zodiac-modifier-delay/blob/main/contracts/Delay.sol#L100-L103).)